draft-ietf-wish-whip.md

```diff
@@ -357,15 +359,15 @@ Both the WHIP client and the WHIP endpoint SHALL support {{!RFC9143}} and use "m
 
 ### Single MediaStream
 
-WHIP only supports a single MediaStream as defined in {{!RFC8830}} and therefore all "m=" sections MUST contain a "msid" attribute with the same value. The MediaStream MUST contain at least one MediaStreamTrack of any media kind and it MUST NOT have two or more than MediaStreamTracks for the same media (audio or video). However, it would be possible for future revisions of this spec to allow more than a single MediaStream or MediaStreamTrack of each media kind, so in order to ensure forward compatibility, if the number of audio and or video MediaStreamTracks or number of MediaStreams are not supported by the WHIP endpoint, it MUST reject the HTTP POST request with a "406 Not Acceptable" error response.
+WHIP only supports a single MediaStream as defined in {{!RFC8830}} and therefore all "m=" sections MUST contain a "msid" attribute with the same value. The MediaStream MUST contain at least one MediaStreamTrack of any media kind and it MUST NOT have two or more than MediaStreamTracks for the same media (audio or video). However, it would be possible for future revisions of this spec to allow more than a single MediaStream or MediaStreamTrack of each media kind, so in order to ensure forward compatibility, if the number of audio and or video MediaStreamTracks or number of MediaStreams are not supported by the WHIP endpoint, it MUST reject the HTTP POST request with an appropiate 4XX error response.
 
```

Comment:
This sounds like another case of a 422 or 400 status code.  I don't think it is unreasonable for you to require the use of one of those HTTP status codes.  As long as the specification doesn't say something like "when you get 400 it means application error X".  The open question remains, if you choose the 400 or 422 status code as the required value, how will the consumer know that they have requested a number of audio/video mediastreamtracks or mediastreams that are not supported?  Do they need to know?  Is it sufficient for them just to know "your request was bad, go use some other tool to debug your request payload"?